heartbeat/monitors/wrappers/monitorstate/esloader.go

```diff
@@ -94,7 +103,8 @@ func MakeESLoader(esc *eslegclient.Connection, indexPattern string, beatLocation
 		sh := stateHits{}
 		err = json.Unmarshal(body, &sh)
 		if err != nil {
-			return nil, fmt.Errorf("could not unmarshal state hits for %s: %w", sf.ID, err)
+			errMsg := fmt.Errorf("could not unmarshal state hits for %s: %w", sf.ID, err).Error()
```

Comment:
ques: why was .Error called here?

Reply:
will check why the log appears "weird":

<img width="307" alt="Screenshot 2024-02-22 at 20 38 04" src="https://github.com/elastic/beats/assets/15065076/11a5b860-9fd2-46e7-bdef-6415d38c7a6c">